### PR TITLE
Create hello.pony

### PR DESCRIPTION
**Pull request by harshraj22**
pony tail ?

### CODE REVIEW

**Review by abhiisshheekk — APPROVED** [review]
Pony ki bony

File: pony/hello.pony [hunk 1]
```diff
@@ -0,0 +1,3 @@
+actor Main
+  new create(env: Env) =>
```
Comment (Manjeetkapil) [inline pony/hello.pony:2]: What is this actor add a multiline comment

**Review by Manjeetkapil — APPROVED** [review]

File: pony/hello.pony (hunk 1 above)
Comment (harshraj22) [reply pony/hello.pony:2]: Multi line sorie shaktimaan

Comment (abhiisshheekk) [reply pony/hello.pony:2]: holi hai!!

**Pull request merged**